Dockerfile

```diff
@@ -1,22 +1,40 @@
-FROM alpine:3.10
+FROM theiaide/theia
 
-ARG KUBECTL_VERSION=1.18.2
-ARG HELM_VERSION=3.1.2
+ARG KUBECTL_VERSION=1.21.1
+ARG HELM_VERSION=3.6.1
+ARG DOCKER_COMPOSE=1.29.2
+ARG TERRAFORM_VERSION=1.0.0
+ARG AZURECLI_VERSION=2.25.0
 
-RUN apk --no-cache add shadow openssh coreutils grep bash lynx sshpass \
-                       openssl curl openssh gettext vim tree tmux git docker-cli==18.09.8-r0 \ 
-		       bash-completion docker-bash-completion git-bash-completion && \
+user root
+RUN sed -i "s/3.11/3.14/" /etc/apk/repositories && \
+    apk --no-cache update && \
+    apk --no-cache add coreutils grep bash curl gettext vim tree git \
+                       docker-cli mysql-client lynx py3-pip figlet \
+                       bash-completion docker-bash-completion git-bash-completion \
+		       py3-yaml py3-pynacl py3-bcrypt py3-cryptography py3-psutil py3-wheel && \
+    # Azure CLI
+    pip3 install azure-cli==${AZURECLI_VERSION} --no-cache-dir && \
+    bash -c "rm -rf /usr/lib/python3.9/site-packages/azure/mgmt/network/v201*" && \
+    bash -c "rm -rf /usr/lib/python3.9/site-packages/azure/mgmt/network/v2020*" && \
+    bash -c "rm -rf /usr/lib/python3.9/site-packages/azure/mgmt/cosmosdb" && \
+    bash -c "rm -rf /usr/lib/python3.9/site-packages/azure/mgmt/iothub" && \
+    bash -c "rm -rf /usr/lib/python3.9/site-packages/azure/mgmt/sql" && \
+    bash -c "rm -rf /usr/lib/python3.9/site-packages/azure/mgmt/web" && \
```

Comment:
Why all the `bash -c`?

Reply:
I think it was because of "shell" is not able to use the wildcard in specific situations. This line is used to shrink the azure cli, which would otherwise like > 300MB